.github/workflows/CI.yml

```diff
@@ -29,41 +29,48 @@ jobs:
         with:
           repository: 'opensearch-project/OpenSearch'
           path: OpenSearch
+          ref: '1.0.0-alpha1'
       - name: Build OpenSearch
         working-directory: ./OpenSearch
-        run: ./gradlew publishToMavenLocal
+        run: ./gradlew publishToMavenLocal -Dbuild.version_qualifier=alpha1 -Dbuild.snapshot=false
 
       # dependencies: common-utils
       - name: Checkout common-utils
         uses: actions/checkout@v2
         with:
-          repository: 'opensearch-project/common-utils'
+          repository: 'dblock/common-utils'
           path: common-utils
+          ref: 1.0.0-alpha1
       - name: Build common-utils
         working-directory: ./common-utils
-        run: ./gradlew publishToMavenLocal
+        run: ./gradlew publishToMavenLocal -Dopensearch.version=1.0.0-alpha1
 
       # dependencies: job-scheduler
       - name: Checkout job-scheduler
         uses: actions/checkout@v2
         with:
-          repository: 'opensearch-project/job-scheduler'
+          repository: 'dblock/job-scheduler'
+          ref: 1.0.0-alpha1
           path: job-scheduler
       - name: Build job-scheduler
         working-directory: ./job-scheduler
-        run: ./gradlew publishToMavenLocal
+        run: ./gradlew publishToMavenLocal -Dopensearch.version=1.0.0-alpha1 -Dbuild.snapshot=false
 
       # anomaly-detection
       - name: Checkout AD
         uses: actions/checkout@v2
 
-      - name: Run test
+      - name: Run Tests
         run: |
-          ./gradlew test
+          ./gradlew test -Dopensearch.version=1.0.0-alpha1 -Dbuild.snapshot=false
+
+      - name: Run Integration Tests
+        run: |
+          ./gradlew integTest -Dopensearch.version=1.0.0-alpha1 -Dbuild.snapshot=false
```

Comment:
I believe for AD `./gradlew build` runs all the tests, check the checkstyle, code coverage etc. 
Instead of both of these, can we just do that?

Reply:
I am seeing a bunch of problems doing that, will do later repo-by-repo.

Reply:
https://github.com/opensearch-project/anomaly-detection/issues/19

Reply:
Sounds good.